Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -131,41 +132,88 @@ internal BlockingRequestHandler(Client client, TimeSpan timeout)
 					{ "batch size", batch.batch.Count }
 				});
 
+				const int MAXIMUM_BACKOFF_DURATION = 10000;
```

Comment:
Can you add a comment indicating this is 10seconds? `const int MAXIMUM_BACKOFF_DURATION = 10000; // 10 seconds`.